java/ql/src/Violations of Best Practice/Undesirable Calls/DefaultToString.qhelp

```diff
@@ -43,10 +43,7 @@ string containing the arguments that were passed when the object <code>p</code>
   J. Bloch, <em>Effective Java (second edition)</em>, Item 10. Addison-Wesley, 2008.
 </li>
 <li>
-  Java 6 API Specification: <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html#toString()">Object.toString()</a>.
-</li>
-<li>
-  Java Language Specification, 3rd ed: <a href="https://docs.oracle.com/javase/specs/jls/se6/html/conversions.html#5.4">5.4 String Conversion</a>.
```

Comment:
Removed this link because it is about String concatenation in genereal and not related to `toString()`.